src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.tsx

```diff
@@ -141,6 +145,13 @@ const ModelledPropertyBuilder: React.FC<ModelledPropertyBuilderProps> = ({
         [onChange]
     );
 
+    const onClickClearButton = useCallback(() => {
+        onChange({
+            expression: ''
+        });
+        setInternalMode(ModelledPropertyBuilderMode.PROPERTY_SELECT);
```

Comment:
do we really want to set the mode back to property selection? for example if I am in intellisense mode, when I click on clear, as a user I wouldn't want the UI switch back to property selection when I click on clear. Let's confirm the expected behavior with Matt

Reply:
@darsney @azimamansuri what do you guys think?

Reply:
ahh i think thats fine actually